src/GrumPHP/Task/PHPLint.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Runner\TaskResultInterface;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * PHP parallel lint task.
+ */
+class PHPLint extends AbstractExternalTask
+{
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'phplint';
+    }
+
+    /**
+     * @return OptionsResolver
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults(array(
+            'jobs' => null,
+            'exclude' => array(),
+        ));
+
+        $resolver->setAllowedTypes('jobs', array('int', 'null'));
+        $resolver->setAllowedTypes('exclude', 'array');
+
+        return $resolver;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function canRunInContext(ContextInterface $context)
+    {
+        return $context instanceof RunContext || $context instanceof GitPreCommitContext;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function run(ContextInterface $context)
+    {
+        $fileNames = array_map(function (\SplFileInfo $file) {
```

Comment:
You don't have to use the array_map method here., just set
```
$files = $context->getFiles()->extensions(array('php'))
```

You could add an additional option `triggered_by` which defaults to ['php'] so that it can be configurable.